src/quacc/atoms/phonons.py

```diff
@@ -60,22 +60,21 @@ def get_phonopy(
     """
     phonopy_kwargs = phonopy_kwargs or {}
 
-    symmetrized_structure = SpacegroupAnalyzer(
-        AseAtomsAdaptor().get_structure(atoms), symprec=symprec
-    ).get_symmetrized_structure()
-
```

Comment:
Was this removed because `Phonopy()` is handling this with `symprec`? We're confident of that?

Reply:
What is this expected to do? I was afraid that it would translate/modify atoms object: this might surprise users that have to place atoms to specific place in unit cells (coulomb corrections, ...) but after trying it seems that it does not do anything? Is it for metadata?

Reply:
The idea is that if you're doing a relaxation on a structure with many degrees of freedom, it almost certainly will be P1 symmetry just due to slight fluctuations. This makes the number of displacements _enormous_ even though it is near a higher symmetry. This line symmetrizes the structure, based on the value of `symprec`, to ensure the phonon displacements are calculated for the symmetrized structure. The default `symprec` is 1e-5, which is a relatively small value. If the structure being passed through the workflow is already at the high symmetry space group or is not within that `symprec` cutoff, there will be no change. If the user doesn't want any symmetrization, they can always toggle `symprec`. 

Now, `symprec` is also passed to `Phonopy`, so my original question was about if Phonopy as already doing this or not. The documentation for the Python API of Phonopy does not make it clear.

Reply:
That said, we can simply leave a note in the docstring and rely on the user to symmetrize their structure before passing it to the flow if we prefer.

Reply:
I would expect Phonopy to make sure of that through the symprec keyword yes:

``` python
slab = bulk("Pt", cubic=True) * (4, 4, 4)
slab.rattle(0.001)

phonon = get_phonopy(slab, supercell_matrix=[1, 1, 1])

print(Symmetry(phonon.get_supercell(), symprec=0.001*3*2).get_international_table())
```
"Fm-3m (225)"

Reply:
We can remove it then